middleware/proxy/upstream.go

```diff
@@ -147,6 +160,15 @@ func parseBlock(c *parse.Dispenser, u *staticUpstream) error {
 			return err
 		}
 		u.MaxFails = int32(n)
+	case "max_conns":
```

Comment:
@mholt This is an awesome feature but not mentioned in the docs: https://caddyserver.com/docs/proxy

How do are the docs updated?